build/Dockerfile.build-container

```
@@ -27,9 +27,12 @@ RUN zypper -n install \
   libboost_thread1_80_0 \
  && zypper clean --all
 
+RUN mkdir -p /radosgw
+ENV LD_LIBRARY_PATH /radosgw:$LD_LIBRARY_PATH
+
 RUN mkdir -p /data
 
-COPY ./bin/radosgw /usr/bin/radosgw
+COPY ./bin/radosgw /radosgw
```

Could you please change that to this to reduce confusion of users that do not have deep knowledge about the `COPY` command and its behaviour.
```suggestion
COPY ./bin/radosgw /radosgw/
```

---

I also don't understand why we're moving binaries to non-obvious locations?

---

> I also don't understand why we're moving binaries to non-obvious locations?

please, read the comment below.